app/src/main/java/com/example/myapplication/SecondFragment.java

```diff
@@ -42,6 +44,7 @@ public View onCreateView(
         binding.speakerUnampCounter.setText("" + MainActivity.speakerUnamp);
         binding.speakerAmpCounter.setText("" + MainActivity.speakerAmp);
         binding.broke.setChecked(MainActivity.broke);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
```

Comment:
This code here can be improved, since the four primatry if statements here are all identical `  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)` and so you could combine these if statements all together into one. Additionally, you use the values `(ColorStateList.valueOf(Color.parseColor("#6750A3"))` and `ColorStateList.valueOf(Color.parseColor("#73C2F0"))` are reused a lot and that's a great indicator to turn them into them into variables that can be simplified so that the code is easier to read

Reply:
This also shows up in SecondFragement2